sgxs-tools/src/sgx_detect/main.rs

```diff
@@ -232,9 +235,13 @@ impl SgxSupport {
             Ok(TimeoutHardError::new(client))
```

Comment:
On line 219 to 224, there is use of certain functions which depend on Linux specific functions.

Reply:
Make a copy of `linux.rs` for windows, and implement Windows functionality there. I think for `rdmsr`, `aesm_status`, `kmod_status` you can always return an error for now (although something should be possible for aesm_status & kmod_status). For read_efi_var you can do [GetFirmwareEnvironmentVariable](https://docs.rs/winapi/0.3.8/x86_64-pc-windows-msvc/winapi/um/winbase/fn.GetFirmwareEnvironmentVariableA.html).